.yamato/upm-ci.yml

```diff
@@ -1,8 +1,7 @@
 editors:
   - version: 2019.4
-  - version: 2020.2
-  - version: 2021.1
-  - version: trunk
+  - version: 2020.3
+  - version: 2021.3
```

Comment:
do we need 2022.2(or 3)

Reply:
I checked [manifest.json of Unity 2022.2 and 2022.1](https://github.cds.internal.unity3d.com/unity/unity/blob/2022.2/staging/External/PackageManager/Editor/manifest.json), its minimally supported version is also 2.3.0. 
It seems 2021.3 is the highest version which still supports `alembic 2.2.2`.